refine_unit_test_code.py

```
@@ -2,19 +2,55 @@
 import ell
 # add importds for rewrite_unit_test_file and rewrite_test_project_file
 from tools import rewrite_unit_test_file
-from llm_clients import topology_client
+from llm_clients import openai_client_for_openrouter, openai_client
```

Ruff (0.8.0)

`llm_clients.openai_client` imported but unused

Remove unused import: `llm_clients.openai_client`

(F401)